src/constants.ts

```diff
@@ -1,5 +1,55 @@
 import * as _ from 'lodash';
 
+_.set(window, 'OPENSHIFT_CONSTANTS.HELP_BASE_URL', "https://docs.openshift.org/latest/");
+var HELP: any = {
```

Comment:
Agreed. I wasn't sure of a good way to do this. These are not important to be the real links as what is set in console will override these. I can leave them out but it was a good way to test from within the showcase app.